services/BasicTypes.proto

```diff
@@ -378,9 +378,11 @@ message NodeAddressBook {
 }
 
 message SemanticVersion {
```

Comment:
Perhaps add a top level comment that states that Hedera follows the semantic versioning spec and provide a link.

Reply:
Done.